instrumentation/opentelemetry-instrumentation-pyramid/src/opentelemetry/instrumentation/pyramid/__init__.py

```diff
@@ -94,9 +92,12 @@
     trace_tween_factory,
 )
 from opentelemetry.instrumentation.pyramid.package import _instruments
-from opentelemetry.instrumentation.pyramid.version import __version__
 from opentelemetry.instrumentation.utils import unwrap
-from opentelemetry.trace import TracerProvider, get_tracer
+
+# test_automatic.TestAutomatic.test_tween_list needs trace_tween_factory to be
+# imported in this module. The next line is necessary to avoid a lint error
+# from importing an unused symbol.
+trace_tween_factory  # pylint: disable=pointless-statement
```

Comment:
any reason to not add pylint disable to the import itself?

Reply:
`pylint` is not the one who fails if there is an unused import, `flake8` is.

Reply:
You can use `# noqa: <error-code>` for flake8. https://flake8.pycqa.org/en/latest/user/violations.html#in-line-ignoring-errors